### ISSUE TITLE
Eye foreign body sensation

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#9870]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA 98957 Gelatinous drop-like corneal dystrophy

**Your nano-attribution (ORCID)**
PMID: 27913443 CLINICAL FEATURES
Patients’ symptoms include severe photophobia, lacrimation, **foreign body sensation**, blepharospasm and a
relatively later onset of progressively deteriorating vision, with manifestation in the first two decades of life.

PMID: 19236704 Gelatinous drop-like corneal dystrophy (GDCD, subepithelial amyloidosis, primary familial amyloidosis, MIM #204870)
Multiple prominent milky-white gelatinous mulberry shaped nodules form beneath the corneal epithelium during the first decade of life GDCD and Other features are severe photophobia, tearing, a **corneal foreign body sensation** and a severe progressive loss
of vision. Fusiform deposits similar to those in lattice corneal dystrophy (LCD) may also form in the deeper stroma.

PMID: 27227392 The phenotype of GDLD can gradually develop over time. In the first decade, there may only be subepithelial
lesions that are similar to band-shaped corneal degeneration, or multiple small grayish epithelial nodules that cause severe
photophobia, tearing, and **foreign body sensation**.

**Comment by pnrobinson** [comment]
Adding as
HP:0034804
Corneal foreign body sensation
def: A perception that an object is in contact with the eye.
